cmd/monaco/download/download_integration_test.go

```diff
@@ -1115,65 +1115,6 @@ func TestDownloadIntegrationDownloadsOnlyAPIsIfConfigured(t *testing.T) {
 	assert.Assert(t, !settingsDownloaded, "Expected no Settings to the downloaded, when onlyAPIs is set")
 }
 
-func TestDownloadIntegrationDownloadsOnlySettingsIfConfigured(t *testing.T) {
```

Comment:
Why is this test removed? Judging by the `DownloadsOnlySettingsIfConfigured` name, this should not be impacted by your changes and is still a valid case to be tested.

Edit: I guess it should use `makeDownloaders` (or an exported method further up that does) to get the correct instantiation now. But that likely goes for all these integration tests. Could they call exported methods or at least production setup methods, rather than building their own downladers and calling the private doDownloadConfigs?

Reply:
As discussed directly: fine by me